imagetest/cmd/wrapper/main.go

```diff
@@ -23,12 +24,21 @@ func main() {
 		log.Fatalf("failed to create cloud storage client: %v", err)
 	}
 	log.Printf("FINISHED-BOOTING")
-	defer func() {
+	defer func(ctx context.Context) {
+		if err := utils.QueryMetadataGuestAttribute(ctx, utils.GuestAttributeTestNamespace, utils.GuestAttributeTestKey, http.MethodPut); err != nil {
+			log.Printf("failed to put test completed key in guest attribute namespace")
+		}
+		log.Printf("successfully placed guest attribute for test completion")
 		for f := 0; f < 5; f++ {
 			log.Printf("FINISHED-TEST")
 			time.Sleep(1 * time.Second)
 		}
-	}()
+		time.Sleep(10 * time.Second)
```

Comment:
I understand why this is needed here, however I believe the actual fix should go in the metadata script runner - we can further discuss it offline.

Reply:
We can get rid of it when we have a fix landed to metadata script runner.